admin/views/school/pupil-register.ejs

```diff
@@ -21,7 +21,7 @@
         <table id='register-pupils' class="spacious">
             <thead>
                 <tr>
-                    <th scope="col">
+                    <th scope="col" style="width: 66%">
```

Comment:
can this be included in css to avoid inline css?
I know we've been doing this for a while but we'll be losing track of what's going on with the styles if we continue with this practice.